src/chrome/content/rules/GMX.xml

```diff
@@ -2,7 +2,6 @@
 
 	For other United Internet coverage, see United-Internet.xml.
 
-
```

Comment:
These whitespaces are not really needed.
I've just removed the double whitespace line. :wink: